mtlpy/views.py

```diff
@@ -94,25 +94,25 @@ def contact(request):
 
 
 def styleguide(request):
-    return render_to_response('styleguide.html', {})
+    return render(request, 'styleguide.html', {})
 
 
 def videos(request):
     videos = get_all_videos(settings.YOUTUBE_API_KEY)
-    return render_to_response('videos.html', {"videos": videos})
+    return render(request, 'videos.html', {"videos": videos})
 
 
 def sponsor_details(request, slug):
     sponsor = get_object_or_404(Sponsor, slug=slug)
-    return render_to_response('sponsor_details.html', {"sponsor": sponsor})
+    return render(request, 'sponsor_details.html', {"sponsor": sponsor})
 
 
 def sponsorship(request):
     sponsors = (Sponsor.objects
                 .filter(frontpage=True)
                 .filter(~Q(partner=True))
```

Comment:
filter not partner == exclude partner == filter partner false

```python

sponsors = Sponsor.objects.filter(frontpage=True, partner=False)
```
:)

(ordering can be defined on the model rather than the view)

Reply:
Effectivement, je prefererais faire ce changement dans une autre PR histoire de minimiser les changements dans celle-ci